values-custom.yaml

```
@@ -2,6 +2,7 @@
 replicaCount:
   frontend: 2
   backend: 2
+  backendutilities: 1
```

Is it possible to use kebab case for readability here (and every other refs) ?🙏

```suggestion
  backend-utilities: 1
```

---

It does not accept "-"